recipe/meta.yaml

```diff
@@ -96,19 +99,30 @@ test:
     {% endfor %}
 
     # Test ncurses library arguments.
-    - pkg-config ncursesw --libs | grep "^-L${PREFIX}/lib -lncursesw -ltinfow$"
-    - pkg-config tinfow --libs | grep "^-L${PREFIX}/lib -ltinfow$"
-    - pkg-config ncurses --libs | grep "^-L${PREFIX}/lib -lncurses -ltinfo$"
-    - pkg-config tinfo --libs | grep "^-L${PREFIX}/lib -ltinfo$"
+    - pkg-config ncursesw --libs | grep "^-L${PREFIX}/lib .* -lncursesw$"
+    - pkg-config tinfow --libs | grep "^-L${PREFIX}/lib .* -ltinfow$"
+    - pkg-config ncurses --libs | grep "^-L${PREFIX}/lib .* -lncursesw$"
+    - pkg-config tinfo --libs | grep "^-L${PREFIX}/lib .* -ltinfow$"
```

Comment:
Why are dropping `tinfo` from the `ncurses` lib checks? Having both of these were important for bioconda users.